db/migrate/20110511215016_translate_page_image_captions.rb

```diff
@@ -1,22 +1,17 @@
 class TranslatePageImageCaptions < ActiveRecord::Migration[4.2]
-  def up
-    add_column Refinery::ImagePage.table_name, :id, :primary_key
```

Comment:
Removing this causes an `Unknown primary key for table refinery_image_pages in model Refinery::ImagePage.` error.